tests/test_report_reader.cpp

```diff
@@ -23,23 +30,105 @@ TEST_CASE("SpikeReader", "[base]") {
             SpikeReader::Population::Sorting::none);
 }
 
-TEST_CASE("ReportReader", "[base]") {
+TEST_CASE("SomaReportReader limits", "[base]") {
+    const SomaReportReader reader("./data/somas.h5");
+
+    auto pop = reader.openPopulation("All");
+
+    // ids out of range
+    // REQUIRE_THROWS(pop.get(Selection({{100, 101}})));
```

Comment:
what s the current behavior ?

Same for `pop.get(Selection({{1, 1}}))`

Reply:
The current behavior is the one from `Selection` ^^

Reply:
perfect

Reply:
[1, 1] is [1]

Reply:
I make it throw again.

Reply:
hum `{{1, 1}}` is refused by `Selection`: `Invalid range: 1-1`